lib/bike_brigade_web/live/rider_live/show.html.heex

```diff
@@ -20,20 +20,20 @@
-            </div>
-            <div class="flex flex-col justify-start text-sm leading-6 sm:text-base md:flex-row md:items-center">
-                <div><Heroicons.Outline.location_marker aria-label="Marker" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 sm:mr-1" /><%= @rider.address %></div>
-                <div><Heroicons.Outline.phone aria-label="Phone" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 md:ml-4 sm:mr-1" /><%= @rider.phone %></div>
-                <div><Heroicons.Outline.mail aria-label="Mail" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 md:ml-4 sm:mr-1" /><%= @rider.email %></div>
-            </div>
-            <div class="flex flex-col justify-start text-sm leading-6 sm:text-base md:flex-row md:items-center">
-                <div><Heroicons.Outline.briefcase aria-label="Briefcase" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 sm:mr-1" />
+            <p class="text-sm leading-6 lg:text-base">
+                <span class="block md:inline-block"><Heroicons.Outline.clipboard_list aria-label="Calendar" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 sm:mr-1" />Signed up on: <%= format_date(@rider.signed_up_on) %></span>
+                <span class="block xl:inline-block"><Heroicons.Outline.fire aria-label="Fire" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 xl:ml-2 sm:mr-1" />Latest activity: <.latest_campaign_info socket={@socket} rider={@rider} /></span>
+            </p>
+            <p class="text-sm leading-6 lg:text-base">
+                <span class="block md:inline-block"><Heroicons.Outline.location_marker aria-label="Marker" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 sm:mr-1" /><%= @rider.address %></span>
+                <span class="block md:inline-block"><Heroicons.Outline.phone aria-label="Phone" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 md:ml-2 sm:mr-1" /><%= @rider.phone %></span>
+                <span class="block xl:inline-block"><Heroicons.Outline.mail aria-label="Mail" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 xl:ml-2 sm:mr-1" /><%= @rider.email %></span>
+            </p>
+            <p class="text-sm leading-6 lg:text-base">
+                <span class="block md:inline-block"><Heroicons.Outline.briefcase aria-label="Briefcase" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 sm:mr-1" />
                     <%= live_redirect @rider.capacity, to: Routes.rider_index_path(@socket, :index, %{capacity: [@rider.capacity]}), class: "link" %>
-                </div>
-                <div class="flex items-center"><Heroicons.Outline.tag aria-label="Tag" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 md:ml-4 sm:mr-1" />
+                </span>
+                <span class="block md:inline-block"><Heroicons.Outline.tag aria-label="Tag" class="inline-flex flex-shrink-0 w-5 h-5 text-gray-400 md:ml-2 sm:mr-1" />
```

Comment:
This is not wrapping the tags right
<img width="338" alt="Screen Shot 2022-02-09 at 2 04 26 PM" src="https://user-images.githubusercontent.com/34720/153272043-1035e5d4-22ac-493a-a54d-97e528c0fd3f.png">

<img width="678" alt="Screen Shot 2022-02-09 at 2 04 39 PM" src="https://user-images.githubusercontent.com/34720/153272061-736fac26-b66e-4910-ba9c-ecd24816062f.png">